__DEFINES/setup.dm

```diff
@@ -529,7 +529,7 @@ var/global/list/bad_changing_colour_ckeys = list()
 #define INV_SLOT_SIGHT "sight_slot"
 #define INV_SLOT_TOOL "tool_slot"
 
-#define IS_MODE_COMPILED(MODE) (ispath(text2path("/datum/game_mode/"+(MODE))))
+//#define IS_MODE_COMPILED(MODE) (ispath(text2path("/datum/gamemode/"+(MODE))))
```

Comment:
Is this indented ?

Reply:
??

Reply:
I mean is something that you want to remove or something that was commented for datums merge-ability